packages/syft/src/syft/service/settings/settings.py

```diff
@@ -1,23 +1,24 @@
 # stdlib
-from typing import Optional
 
 # relative
 from ...serde.serializable import serializable
+from ...types.syft_object import PartialSyftObject
 from ...types.syft_object import SYFT_OBJECT_VERSION_1
 from ...types.syft_object import SyftObject
 from ...types.uid import UID
 
 
 @serializable()
-class NodeSettingsUpdate(SyftObject):
+class NodeSettingsUpdate(PartialSyftObject):
```

Comment:
Q: Interesting , what is PartialSyftObject, how does it help this context ?

Reply:
So, PartialSyftObject class, allows us to declare attributes without marking them Optional all the time. This allows us to havev a distinction between values when they are None and when the user has not set them.
This means, if I don't provide description in a Project object, then it is value is marked as `Empty` instead of None. 

Refer to this issue for more detail:
https://github.com/OpenMined/Heartbeat/issues/117